### ISSUE TITLE
link to example doesn't work - Failure of Success Criterion 1.3.4 due to showing a message asking to reorient device

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1416]
in

https://www.w3.org/WAI/WCAG22/Techniques/failures/F100

as well as 

https://www.w3.org/WAI/WCAG21/Techniques/failures/F100

The link to the example: "Working example: [Show message asking to reorient device](https://www.w3.org/WAI/WCAG22/working-examples/css-reorient/)" is broken

**Comment by alastc** [comment]
Fixed in master.
